libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -1700,6 +1788,8 @@ CK_DECLARE_FUNCTION( CK_RV, C_GetMechanismInfo )( CK_SLOT_ID slotID,
     if( pInfo == NULL )
     {
         xResult = CKR_ARGUMENTS_BAD;
+        LogError( ( "Failed to get mechanism info. pInfo was NULL. Expected a "
+                    "pointer to a valid CK_MECHANISM_INFO struct." ) );
     }
```

Comment:
Outside of the scope of this PR, but for readability I would move 1788-1793 to 1811 and change 1812 to an `else`.

Reply:
Noted, I will update this